docs/getting_started.md

```diff
@@ -22,13 +22,13 @@ Textual requires Python 3.7 or later (if you have a choice, pick the most recent
 
 You can install Textual via PyPI.
 
-If you plan on developing Textual apps, then you should install `textual[dev]`. The `[dev]` part installs a few extra dependencies for development.
+If you plan on developing Textual apps, then you should install `textual-dev`; this installs a few extra dependencies for development.
```

Comment:
I think we should spell out the two imports in two code fences.

Start with a suggestion to `pip install textual`. Then a recommendation to `pip install textual-dev` for the dev tools.

I know that technically only `textual-dev` is required, but I think this will help to keep the two packages distinct in the developers mind. We want to avoid devs thinking they need to add `textual-dev` to their package dependancies.